src/components/common/NameTranslations.vue

```diff
@@ -6,7 +6,7 @@
     />
 
     <div
-      class="section-container pt-0"
+      class="section-container"
```

Comment:
(Sub)components should not specify the whitespace around them -- so that the parent components can specify it instead.

Reply:
NOTE: I'm reworking this.

Reply:
Outdated. Closing.